src/main/resources/swagger/api-docs.yaml

```diff
@@ -1695,6 +1695,131 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/resources/v2/{resourceTypeName}/{resourceId}/policies/{policyName}/memberPolicies/{memberResourceTypeName}/{memberResourceId}/{memberPolicyName}:
+    put:
+      tags:
+        - Resources
+      summary: Add a user to a policy by role on another resource
+      operationId: addUserToPolicyByRoleV2
+      parameters:
+        - name: resourceTypeName
+          in: path
+          description: Type of resource
+          required: true
+          schema:
+            type: string
+        - name: resourceId
+          in: path
+          description: Id of resource
+          required: true
+          schema:
+            type: string
+        - name: policyName
+          in: path
+          description: Name of the policy
+          required: true
+          schema:
+            type: string
+        - name: memberResourceTypeName
+          in: path
+          description: Type of resource for the member policy
+          required: true
+          schema:
+            type: string
+        - name: memberResourceId
+          in: path
+          description: Resource id for the member policy
+          required: true
+          schema:
+            type: string
+        - name: memberPolicyName
+          in: path
+          description: Member policy name
+          required: true
+          schema:
+            type: string
+      responses:
+        204:
+          description: Successfully added a user to the policy
+          content: { }
+        400:
+          description: email is not found
```

Comment:
The nittiest of all nits, but for consistency/"professionalism" maybe capitalize email.

Reply:
Ah! This is a mistake- there is no email validation and no 400 error case at all. I copied this from the "memberEmails" API case, but it doesn't apply here